test/testcafe/spec/SessionStorage_spec.js

```diff
@@ -154,7 +156,7 @@ test.requestHooks(identifyChallengeMock)('shall clear session.stateHandle when c
 
   // Go back to Identify page after sign-out
   await t.expect(identityPage.form.getTitle()).eql('Sign In');
-  await t.expect(getStateHandleFromSessionStorage()).eql(null);
+  await t.expect(getStateHandleFromSessionStorage()).notEql(xhrEmailVerification.stateHandle);
```

Comment:
This test is asserting the `stateHandle` is empty, so we're breaking existing behavior with this change.

What do we expect this value to be now?

Reply:
Because of starting new login flow after cancel (the purpose of this PR), state handle should be a new one.

Reply:
Right, my comment was mostly focused on ensuring the assertions match the description.

Let's make sure we update the test description and have an assertion for the new `stateHandle` we'd expect.